### PR TITLE
Study burst view

### PR DESCRIPTION
**Pull request by joshbruhin**
- Add 'StudyBurstViewController' class and storyboard definition; there are several TODO items remaining in the class
- Add 'StudyBurstTaskBrowserViewController' subclass of 'TaskBrowserViewController'
- Refactor several items in 'TaskBrowserViewController' to accommodate new subclass
- Add several new task icon images, replace several others and delete a couple
- A couple minor tweaks in 'TodayViewController' storyboard definition, such as making title and text labels scale their font to fit longer content

### CODE REVIEW

File: mPower2/mPower2/View Controllers/TodayViewController.swift [hunk 1]
```diff
@@ -523,14 +502,64 @@ class TodayTableViewCell: UITableViewCell {
 }
 
 extension RSDTaskInfoObject {
```
Comment (syoung-smallwisdom) [inline mPower2/mPower2/View Controllers/TodayViewController.swift]: Why is this  extension applied to RSDTaskInfoObject whereas the other is applied to RSDTaskInfo?

**Comment by syoung-smallwisdom** [comment]
@joshbruhin 
There was a conflict with my pull request. I merged that change. Can you please add that to this PR? Sorry about the confusion. `isCompleted` checkmark should only be applied to the Measuring tasks.

Thanks,
S

File: mPower2/mPower2/View Controllers/TodayViewController.swift (hunk 1 above)
Comment (joshbruhin) [reply mPower2/mPower2/View Controllers/TodayViewController.swift]: @syoung-smallwisdom not sure, but I removed the one for 'pluralTerm' as it wasn't being used.

**Comment by joshbruhin** [comment]
@syoung-smallwisdom merged in your change for isCompleted. Should be good to go

**Pull request merged**